_harp/services/ratecard.md

```diff
@@ -0,0 +1,16 @@
+# Rate card
+
+|    |    |
+|----|---:|
+|Graduate developer team|£500/week|
+|Graduate developer - follow-up work|£80/day|
+|Apprentice developer|£100/day|
```

Comment:
And I believe the principle of starting rates of £10hr and £15/hr for apprentices and devs has already been agreed at previous meetings (even though many have been working at 12.50/hr in recent weeks).